src/calendar/index.js

```diff
@@ -400,6 +403,10 @@ class Calendar extends Component {
             : <CalendarHeader {...headerProps}/>
           }
           <View style={this.style.monthView}>{weeks}</View>
+          { CustomFooter
+            ? <CustomFooter {...headerProps}/>
+            : null
```

Comment:
Why passing the 'headerProps' to the footer? Also, you don't need a trinary here, a simple {CustomFooter && < CustomFooter/>} should do the trick.

Reply:
Why not use a custom render function? I mean, a 'renderFooter' prop that will be called here and return the element, instead of passing the element?